bigquery/tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
```

Comment:
I will merge necessary bit to the top level tox.ini, so maybe you can remove it?